Makefile

```diff
@@ -31,7 +31,7 @@ $(SRC_DIR)/%.ttf: $(SRC_DIR)/%.ufo $(SRC_DIR)/%.ufo/*.plist \
 $(BUILD_DIR)/%.ttf: $(SRC_DIR)/%.ttf
 	@mkdir -p $(BUILD_DIR)
 	python -m vttLib compile --ship $< $@
-	tools/update-hinted-metrics.sh $@
+	bash tools/update-hinted-metrics.sh $@
```

Comment:
this is probably because "." is not in the `$PATH`, so needs:

  ./tools/update-hinted-metrics.sh

Scripts frequently do not need `bash`, but can be run under `sh`, which on Debian/Ubuntu is a simpler and faster shell implementation.